script/validate/vendor

```diff
@@ -37,13 +37,16 @@ DIFF_PATH="vendor/ go.mod go.sum"
 #
 # shellcheck disable=SC2046
 DIFF=$(git status --porcelain -- $DIFF_PATH)
+DIFF_CONTENT=$(git diff -- $DIFF_PATH)
```

Comment:
Minor: Since `DIFF_CONTENT` isn't used unless `DIFF` is non-empty, would it make sense to move this assignment inside the `if` statement below?

Reply:
Actually, disregard that; if `DIFF` is empty, the return value of `git diff --` is going to be too, so there isn't really any significant optimization to be gained by moving that declaration.